Makefile

```diff
@@ -6,5 +6,5 @@ build:
 test:
 	pip3 install -r test_requirements.txt
 	flake8 --exclude ./lib/*
-	python3 -m unittest tests/*.py
+	pytest --cov app
```

Comment:
pytest -v --cov app will produce a more verbose output of individual files if thats the direction we want to move in. http://pythontesting.net/framework/pytest/pytest-introduction/